src/main/resources/assets/computercraft/lang/de_de.lang

```diff
@@ -43,32 +43,144 @@ chat.computercraft.wired_modem.peripheral_connected=Peripheriegerät "%s" mit de
+commands.computercraft.track.stop.action=Klicke um die Aufzeichnung zu stoppen
+commands.computercraft.track.stop.not_enabled=Momentan werden keine Computer aufgezeichnet
+
+commands.computercraft.track.dump.synopsis=Gibt die aktuellen Ergebnisse der Aufzeichnung aus
+commands.computercraft.track.dump.desc=Gibt die aktuellen Ergebnisse der Aufzeichnung aus.
+commands.computercraft.track.dump.usage=[Art]
+commands.computercraft.track.dump.no_timings=Keine Zeitangaben verfügbar
+commands.computercraft.track.dump.no_field=Unbekanntes Feld '%s'
+commands.computercraft.track.dump.computer=Computer
+
+commands.computercraft.reload.synopsis=List die Konfigurationsdatei von ComputerCraft neu ein
+commands.computercraft.reload.desc=List die Konfigurationsdatei von ComputerCraft neu ein
+commands.computercraft.reload.usage=
+commands.computercraft.reload.done=Die Konfigurationsdatei wurde erfolgreich neu eingelesen
+
+commands.computercraft.queue.synopsis=Sendet ein computer_command Event an einen Befehlscomputer
+commands.computercraft.queue.desc=Sendet ein computer_command Event zusammen mit optionalen Argumenten an einen Befehlscomputer. Dieser Befehl wurde als eine Computerfreundliche Version von /trigger für Mapdesigner designed. Jeder Spieler kann diesen Befehl ausführen, weshalb er sich perfekt für ein Klickevent von z.B. Schildern oder Büchern eignet.
+commands.computercraft.queue.usage=<ID> [Argumente...]
+
+commands.computercraft.generic.no_position=<Keine Position>
+commands.computercraft.generic.position=%s, %s, %s
+commands.computercraft.generic.yes=J
+commands.computercraft.generic.no=N
+commands.computercraft.generic.exception=Unbehandelte Ausnahme (%s)
+commands.computercraft.generic.additional_rows=%d zusätzliche Zeilen…
+
+commands.computercraft.argument.no_matching=kein Computer passt auf '%s'
+commands.computercraft.argument.many_matching=Mehre Computer passen auf '%s' (Instanzen %s)
+commands.computercraft.argument.not_number='%s' ist keine Zahl
 
 # Names for the various tracking fields.
+tracking_field.computercraft.tasks.name=Aufgaben
+tracking_field.computercraft.total.name=Gesamtzeit
+tracking_field.computercraft.average.name=Durchschnittliche Zeit
+tracking_field.computercraft.max.name=Maximale Zeit
+
+tracking_field.computercraft.server_count.name=Anzahl der Server-Tasks
+tracking_field.computercraft.server_time.name=Server-Task-Zeit
+
+tracking_field.computercraft.peripheral.name=Peripheriegeräte Aufrufe
+tracking_field.computercraft.fs.name=Dateisystem Operationen
+tracking_field.computercraft.turtle.name=Turtle Operationen
+
+tracking_field.computercraft.http.name=HTTP-Requests
+tracking_field.computercraft.http_upload.name=HTTP Upload
+tracking_field.computercraft.http_download.name=HTTT Download
+
+tracking_field.computercraft.websocket_incoming.name=Websocket eingehend
+tracking_field.computercraft.websocket_outgoing.name=Websocket ausgehend
+
+tracking_field.computercraft.coroutines_created.name=Coroutines erstellt
```

Comment:
Koroutine is a word

Reply:
Bei LUA wird auch in Deutsch eigentlich immer das englishe Word verwendet. Ich habe dort noch nie das deutsche Wort gelesen.

Reply:
![](https://shitty.download/n6Vk8.png)

Reply:
Als Informatiker würde ich Koroutine sagen, aber ist ja eigentlich egal.

Reply:
Ich hab das Wort zwar noch nie im Bezug auf LUA gehört, habe es aber mal trotzdem abgeändert.

Reply:
Lua ist kein Akronym, es wird nicht LUA geschrieben.